drivers/scheduler/k8s/k8s.go

```diff
@@ -6553,3 +6553,12 @@ func init() {
 	k := &K8s{}
 	scheduler.Register(SchedName, k)
 }
+
+// ClusterVersion returns the cluster version of the kubernetes cluster as a string (like "1.23.0")
+func ClusterVersion() (string, error) {
```

Comment:
There is already a function that returns the kubernetes version `d.schedOps.GetKubernetesVersion()`

Reply:
This also return type `*version.Info` in this function `ClusterVersion()`  I'm converting that `*version.Info` type to string in format  "1.23.0"